podcasts/AppDelegate.swift

```diff
@@ -299,7 +298,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     private func updateRemoteFeatureFlags() {
         #if !DEBUG
         do {
-            try FeatureFlagOverrideStore().override(FeatureFlag.patron, withValue: Settings.patronEnabled)
```

Comment:
There are still some more code we can remove:

- https://github.com/Automattic/pocket-casts-ios/blob/trunk/podcasts/Settings.swift#L897
- https://github.com/Automattic/pocket-casts-ios/blob/trunk/podcasts/Constants.swift#L327-L328